docs/components/Gallery/Web.stories.tsx

```diff
@@ -22,71 +22,80 @@ const files = [
     type: "image/png",
     size: 213402324,
     progress: 1,
-    src: "https://picsum.photos/250",
+    thumbnailSrc: "https://picsum.photos/250",
```

Comment:
Strangly in story book, under canvas I see the correct behaviour where the thumbnail image and the actual image loaded dont match but when I go to the code section, I get the same photo for both despite the code saying its two seperate URL's. I dont think there is a problem other than it not reflecting what is really being used.

Reply:
the code tab is loading prod version of atlantis. we really need to throw a banner there 😆

Reply:
Ohhhh okay haha thanks I did not know this!